crates/typst/src/eval/markup.rs

```diff
@@ -52,7 +55,22 @@ fn eval_markup<'a>(
                     if let Some(elem) =
                         seq.iter_mut().rev().find(|node| !node.can::<dyn Unlabellable>())
                     {
+                        if let Some(prev_label) = elem.label() {
+                            vm.engine.sink.warn(warning!(
+                                prev_label_expr.unwrap().span(),
```

Comment:
I think this unwrap can panic in practice. The previous label needn't come from the same content block, it could have been applied anywhere in a different function call.

Reply:
Not necessarily, consider this: `#[#block()<a>]<b>`

Reply:
It would break hash and equality identity. There would be ways around this I guess, but I don't think it would be a good change. It also increases the size a lot for a type that is intentionally very light-weight (just u32).